airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/LowCreditBalanceHint/LowCreditBalanceHint.tsx

```diff
@@ -0,0 +1,37 @@
+import { faCreditCard, faWarning } from "@fortawesome/free-solid-svg-icons";
+import classNames from "classnames";
+import { FormattedMessage } from "react-intl";
+
+import { InfoBox } from "components/ui/InfoBox";
+
+import { useGetCloudWorkspace } from "packages/cloud/services/workspaces/CloudWorkspacesService";
+import { useCurrentWorkspace } from "services/workspaces/WorkspacesService";
+
+// eslint-disable-next-line css-modules/no-unused-class
+import styles from "./LowCreditBalanceHint.module.scss";
```

Comment:
Since I'm accessing classes dynamically, the linter can't determine if classes on `LowCreditBalanceHint.module.scss` are read, therefore I need to add ignore it.

Reply:
We usually try to create a map to status to classname, so we don't need to disable linting here, e.g.

```js
const statusStyles = {
  "zeroBalance": styles.zeroBalance,
  "lowBalance": styles.lowBalance,
};
```

Since this is only two options, I feel we can inline this more nicely into the `className` call in line 30 directly:

```js
classNames(styles.container, {
  [styles.zeroBalance]: status === "zeroBalance",
  [styles.lowBalance]: status === "lowBalance",
});
```

This way we get proper checking on the used and unused classnames still from the linter.